### PR TITLE
Redirects all requests to jetty port 80 to the secure port(default 443).

### PR DESCRIPTION
**Pull request by damencho**

### CODE REVIEW

File: src/main/java/org/jitsi/videobridge/rest/PublicRESTBundleActivator.java [hunk 1]
```diff
@@ -64,6 +64,8 @@
     public static final String JETTY_CORS_ALLOWED_ORIGINS
         = ".jetty.cors.allowedOrigins";
 
+    public static final String JETTY_TLS_PORT_PNAME = ".jetty.tls.port";
```
Comment (bgrozev) [inline src/main/java/org/jitsi/videobridge/rest/PublicRESTBundleActivator.java]: Why are we not re-using the existing constant for this (same name in super)?

**Comment by bgrozev** [comment]
What happens if the bridge fails to bind on 80? Just a warning in the logs?

File: src/main/java/org/jitsi/videobridge/rest/PublicRESTBundleActivator.java (hunk 1 above)
Comment (damencho) [reply src/main/java/org/jitsi/videobridge/rest/PublicRESTBundleActivator.java]: Ups, I missed it. Fixing it.

**Comment by damencho** [comment]
> What happens if the bridge fails to bind on 80? Just a warning in the logs?

And two stack traces, but they are coming from the abstract and we cannot skip them. But other than that everything continues as usual.

File: src/main/java/org/jitsi/videobridge/rest/PublicClearPortRedirectBundleActivator.java [hunk 2]
```diff
@@ -0,0 +1,173 @@
+/*
+ * Copyright @ 2015 Atlassian Pty Ltd
```
Comment (saghul) [inline src/main/java/org/jitsi/videobridge/rest/PublicClearPortRedirectBundleActivator.java]: 2018

Comment (damencho) [reply src/main/java/org/jitsi/videobridge/rest/PublicClearPortRedirectBundleActivator.java]: eee copy paste should automatically replace this :)

**Pull request merged**